server/core/runtime/policy/conftest_executor.go

```diff
@@ -62,62 +55,61 @@ func NewConfTestExecutor(creator githubapp.ClientCreator, org string) *ConfTestE
 // Run performs conftest policy tests against changes and fails if any policy does not pass. It also runs an all-or-nothing
 // filter that will filter out all policy failures based on the filter criteria.
 func (c *ConfTestExecutor) Run(_ context.Context, prjCtx command.ProjectContext, executablePath string, envs map[string]string, workdir string, extraArgs []string) (string, error) {
-	var policyArgs []Arg
 	var policyNames []string
 	var failedPolicies []valid.PolicySet
+	var totalCmdOutput string
+	var policyErr error
+
 	inputFile := filepath.Join(workdir, prjCtx.GetShowResultFileName())
 	scope := prjCtx.Scope.SubScope(conftestScope)
 
 	for _, policySet := range prjCtx.PolicySets.PolicySets {
-		path, err := c.SourceResolver.Resolve(policySet)
-		// Let's not fail the whole step because of a single failure. Log and fail silently
-		if err != nil {
-			prjCtx.Log.ErrorContext(prjCtx.RequestCtx, fmt.Sprintf("Error resolving policyset %s. err: %s", policySet.Name, err.Error()))
-			continue
+		var policyArgs []Arg
```

Comment:
nit: thoughts on declaring this outside the for loop and resetting it to an empty list for every iteration?

Reply:
Personally, I feel like this is slightly cleaner since it's one line and has an equivalent meaning.